widgets/ItemButton.lua

```diff
@@ -639,7 +655,8 @@ end
 function stackProto:UpdateCount()
 	local count = 0
 	for slotId in pairs(self.slots) do
-		count = count + (select(2, GetContainerItemInfo(GetBagSlotFromId(slotId))) or 1)
+		itemCount = SafeGetItem(GetContainerItemInfo(GetBagSlotFromId(slotId)), "stackCount") or 1
+s
```

Comment:
`s` mistake?

Reply:
Nice catch! I removed the whole line ... now it is fixed